src/components/IncomingObjects/IncomingObjects.js

```diff
@@ -29,13 +35,24 @@ const IncomingObjects = ({ handleNetworkErrors }) => {
   }
 
   return(
-    <>
-    </>
+    <section className='general-container'>
+      <h2 className='events-subtitle'>Celestial Appointments</h2>
+      <p className='list-action'>Mark your cosmic calendar!</p>
+      <p className='list-explanation' >Get ready for upcoming celestial close approaches, click on any event to get more information. </p>
+    
+      
+      <div className='upcoming-container'>
+        <p className='upcoming-item list-leader'>Date & Time</p>  
+        <p className='upcoming-item list-leader'>Designation</p>
+      </div>
+        {objectEventList}
+    </section>
```

Comment:
Will there be a save button added here?
I was also feeling like the data visually looks "merged". If you could divide the individual events visually by maybe having a border if it's visually in sync with your vision? (totally get it if not). 

Also, could an explanation of "designation" be present here as well?

Reply:
Thanks! I'll be keeping this in mind :)

Reply:
Thanks for the suggestions! I'll be keeping that in mind after adding the save button.